actions/run_action.go

```diff
@@ -86,14 +90,19 @@ func (run *RunAction) doRun(context debos.DebosContext) error {
 	if run.Script != "" {
 		run.Script = debos.CleanPathAt(run.Script, context.RecipeDir)
 		if run.Chroot {
-			cmd.AddBindMount(path.Dir(run.Script), "/script")
-			cmdline = []string{fmt.Sprintf("/script/%s", path.Base(run.Script))}
+			scriptpath, err := debos.GetCommandFromString(run.Script)
+			if err != nil {
+				return err
+			}
+			scriptpath = path.Dir(scriptpath)
+			cmd.AddBindMount(scriptpath, "/script")
+			cmdline = []string{strings.Replace(run.Script, scriptpath, "/script", 1)}
```

Comment:
That will break if the directory name of the script and some other parameters overlap; please make sure you only replace the script name not the full command line

Reply:
There is a limit '1' for replacement in `strings.Replace(..,..,1)`